jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/writer/Iso88591Writer.java

```diff
@@ -11,25 +11,28 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.ee9.nested;
+package org.eclipse.jetty.io.writer;
 
 import java.io.IOException;
+import java.io.OutputStream;
 
 /**
- *
+ * An implementation of {@link AbstractOutputStreamWriter} for
+ * optimal ISO-8859-1 conversion.
+ * The ISO-8859-1 encoding is done by this class and no additional
+ * buffers or Writers are used.
  */
-public class Iso88591HttpWriter extends HttpWriter
+public class Iso88591Writer extends AbstractOutputStreamWriter
```

Comment:
This class should be internal or package local, or an private inner class of `AbstractOutputStreamWriter`.